src/DIRAC/WorkloadManagementSystem/JobWrapper/test/Test_JobWrapper.py

```diff
@@ -80,22 +80,22 @@ def test_execute(mocker):
   print('jw.execute() returns', str(res))
   assert res['OK']
 
-  shutil.copy('WorkloadManagementSystem/JobWrapper/test/script-OK.sh', 'script-OK.sh')
+  shutil.copy('src/DIRAC/WorkloadManagementSystem/JobWrapper/test/script-OK.sh', 'script-OK.sh')
   jw = JobWrapper()
   jw.jobArgs = {'Executable': 'script-OK.sh'}
   res = jw.execute()
   assert res['OK']
   os.remove('script-OK.sh')
 
-  shutil.copy('WorkloadManagementSystem/JobWrapper/test/script.sh', 'script.sh')
+  shutil.copy('src/DIRAC/WorkloadManagementSystem/JobWrapper/test/script.sh', 'script.sh')
   jw = JobWrapper()
   jw.jobArgs = {'Executable': 'script.sh', 'Arguments': '111'}
   res = jw.execute()
   assert res['OK']  # In this case the application finished with errors,
   # but the JobWrapper executed successfully
   os.remove('script.sh')
 
-  shutil.copy('WorkloadManagementSystem/JobWrapper/test/script-RESC.sh', 'script-RESC.sh')  # this will reschedule
+  shutil.copy('src/DIRAC/WorkloadManagementSystem/JobWrapper/test/script-RESC.sh', 'script-RESC.sh')  # this will reschedule
```

Comment:
To make pylint happy:
```suggestion
  # this will reschedule
  shutil.copy('src/DIRAC/WorkloadManagementSystem/JobWrapper/test/script-RESC.sh', 'script-RESC.sh')
```

Reply:
I did it already